targets/hudl/main.cpp

```diff
@@ -2,29 +2,29 @@
  * This is the main code running on the HUDL responsible for displaying
  * information that other boards that broadcast through the CAN network.
  */
-#include <EVT/dev/Timer.hpp>
-#include <EVT/io/CANopen.hpp>
-#include <EVT/io/GPIO.hpp>
-#include <EVT/io/pin.hpp>
-#include <EVT/manager.hpp>
-#include <EVT/utils/log.hpp>
-#include <EVT/utils/time.hpp>
+#include <core/dev/Timer.hpp>
+#include <core/io/CANopen.hpp>
+#include <core/io/GPIO.hpp>
+#include <core/io/pin.hpp>
+#include <core/manager.hpp>
+#include <core/utils/log.hpp>
+#include <core/utils/time.hpp>
 
 #include <dev/HUDL.hpp>
 
-namespace IO = EVT::core::IO;
-namespace DEV = EVT::core::DEV;
-namespace time = EVT::core::time;
-namespace log = EVT::core::log;
+namespace io   = core::io;
+namespace dev  = core::dev;
+namespace time = core::time;
+namespace log  = core::log;
 using namespace std;
```

Comment:
I would maybe separate out the std namespace since it is using it and not redefining it.